applications/nyaml/NXstxm.yaml

```diff
@@ -69,6 +69,7 @@ NXstxm(NXobject):
                                  of 3 dimensions, with the first being the set of scan points and the latter
                                  two being the x- and y- extent of the detector.
                                      NOTE: For each dimension > 1 there must exist a dim section such as:
```

Comment:
This is comment that is neither required nor formatted fine, should be changed in NXstxm